storage_service/static/js/project.js

```diff
@@ -1,5 +1,30 @@
 
-
 $(document).ready(function() {
-    $('.datatable').dataTable();
+    $('.datatable').dataTable({
+      // List of language strings from https://datatables.net/reference/option/language
+      oLanguage: {
```

Comment:
I guess Storage could be upgraded to 1.10 here too, but I'd feel happier treating it as a separate task :)